### DISCUSSION TITLE
SMS authentication not available for Nepal

### DISCUSSION THREAD

**Discussion by ArunBikramkhatri** [community/community#122918] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

As github suggest to add 2FA , I found that SMS authentication is not available for country Nepal . So I request to add  sms authentication for Nepal

**Comment by bryandbernhardt** [comment]
GitHub’s two-factor authentication (2FA) via SMS is only supported in certain countries due to delivery success rates. Unfortunately, Nepal is currently not on the list of countries where SMS authentication is supported. GitHub periodically updates this list, so there’s a possibility that SMS authentication for Nepal could be added in the future if the delivery rates improve.

In the meantime, if you’re from Nepal and need to set up 2FA, GitHub recommends using a Time-based One-Time Password (TOTP) mobile application like Google Authenticator or Authy. These apps generate a one-time password that you can use to authenticate yourself to GitHub. For more information on how to configure 2FA using a TOTP application, you can review GitHub’s documentation.

If you feel strongly about this issue, you might consider reaching out to GitHub support directly to express your request. They may be able to provide more information on any potential plans to include Nepal in the SMS authentication list or suggest alternative methods to secure your account.

**Comment by leofun01** [comment]
It would be better to request make 2FA optional.
